.github/workflows/clang-tidy.yaml

```diff
@@ -10,9 +10,6 @@ on:
       - 'docs/**'
       - '.github/**'
   workflow_dispatch:
-  schedule:
```

Comment:
Looks reasonable to me; I suggest modifying the subject line though: drop the word `with`. Proposed subject: `github: do not run clang-tidy as a cron job`.

Reply:
resolved in v2